h2o-algos/src/main/java/hex/glrm/GLRM.java

```diff
@@ -397,6 +397,13 @@ public final boolean hasClosedForm(long na_cnt) {
         InitialXSVD xtsk = new InitialXSVD(dsqrt, _parms._k, _ncolA, _ncolX);
         xtsk.doAll(fullFrm);
 
+        // fix for PUBDEV-3528.  Copy content of fullFrm into dfrm for X, W
```

Comment:
Looks like the vectors are already shared between the frames, is this really needed?
The firs replace looks like a no-op to me. Ran it in debugger and it looks like a no-op (i.e. replace the vec by exactly the same vec cause the vecs are shared.)
The second replace seems suspicious - are we replacing two vecs in  he frame with the same vec?